strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/booters/StorageBooter.java

```diff
@@ -60,65 +60,34 @@ public StorageBooter()
     public void initialize()
             throws IOException, RepositoryManagementStrategyException
     {
-        lockFile = Paths.get(propertiesBooter.getVaultDirectory()).resolve("storage-booter.lock");
+        lock = hazelcastInstance.getLock("StorageBooterLock");
 
-        if (!lockExists())
+        if (lock.tryLock())
         {
-            createLockFile();
-            
-            final Configuration configuration = configurationManager.getConfiguration();
+            try {
+                final Configuration configuration = configurationManager.getConfiguration();
```

Comment:
Cosmetic: Please follow our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html). Braces on new line, please.

Reply:
Sure I will do a commit of formatting, right away.